infra/base-images/base-builder/compile_centipede

```diff
@@ -24,7 +24,7 @@ fi
 cp "$CENTIPEDE_BIN_DIR/libcentipede_runner.pic.a" "$LIB_FUZZING_ENGINE"
 
 export CENTIPEDE_FLAGS=`cat "$SRC/fuzztest/centipede/clang-flags.txt" | tr '\n' ' '`
-export LIBRARIES_FLAGS="-Wno-unused-command-line-argument -Wl,-ldl -Wl,-lrt -Wl,-lpthread -Wl,$SRC/fuzztest/centipede/weak.o"
+export LIBRARIES_FLAGS="-Wno-unused-command-line-argument -Wl,-lstdc++ -Wl,-ldl -Wl,-lrt -Wl,-lpthread -Wl,$SRC/fuzztest/centipede/weak.o"
```

Comment:
@jonathanmetzman Would it be a bad idea to add `stdlibc++` here?
I recall we do not use `stdlibc++` and do not expect projects to use it, but it is required to fix `skcms` build failure.

Reply:
I think we definitely should not add -lstdc++. We already link against libc++
Let me check on the skcms problem.

Reply:
Thanks!